### ISSUE TITLE
Binding disabled attribute generates invalid markup

### ISSUE THREAD

**Issue by andhofmt** [emberjs/ember.js#14436]
Using `attributeBindings[]` from within an Ember.Component object should allow binding of attributes to the component. With most attributes of the form `<attribute-name>=<attribute-value>` the bindings work correctly. 

However, the disabled attribute does not function the same way. Disabled elements should be generated as `<a disabled></a>` rather than `<a disabled="true"></a>` or `<a disabled="false"></a>`

tl;dr, disabled attribute has no value - yet Ember generates it as such.

**Comment by rwjblue** [comment]
Thanks for opening an issue! It's going to be somewhat hard to track this down without a reproduction, can you put together an ember-twiddle.com demonstrating the issue you described?

**Comment by rwjblue** [comment]
I suspect that you may be passing a string to the attribute binding (not an actual boolean value), but really need a demo/snippet to understand what is going on here...

**Comment by pixelhandler** [comment]
@andhofmt See https://github.com/emberjs/ember.js/issues/11828#issuecomment-124768578 I'm curious if the issue you are mentioning is related.

**Comment by Serabe** [comment]
@andhofmt can you follow on this issue, please? Thank you!

**Comment by Serabe** [comment]
Closing this according to [our triage policy](https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md#triaging-policy). Please, provide a reproduction so we can follow up on this.

Thank you!
